src/assets/sass/protocol/components/_hero.scss

```diff
@@ -0,0 +1,259 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+// Page hero section - should usually appear at the top of a page
+
+// - The content is centered by default without a hero image.
+// - Add the 'mzp-has-image' class to adjust the layout for a hero image (body left, image right).
+// - Add the 'mzp-l-reverse' class to reverse the layout (image left, body right).
+// - With an image, content is still centered in small viewports, with the image below the body.
+// - Add the 'mzp-t-firefox' class for Firefox branding (changing font to Open Sans).
+// - Add the 'mzp-t-dark' class to invert text colors for dark backgrounds.
+
+.mzp-c-hero {
+    padding-bottom: $spacing-lg;
+    position: relative;
+    text-align: center;
+
+    // Product logos
+    &.mzp-t-product-firefox,
+    &.mzp-t-product-beta,
+    &.mzp-t-product-developer,
+    &.mzp-t-product-nightly,
+    &.mzp-t-product-focus {
+        .mzp-c-hero-title {
+            @include background-size(80px 80px);
+            background-position: top center;
+            background-repeat: no-repeat;
+            padding: (80px + $padding-lg) 0 0 0;
+        }
+    }
+
+    // Dark theme
+    &.mzp-t-dark,
+    .mzp-t-dark & {
+        background-color: $color-gray-80;
+        color: $color-white;
+
+        .mzp-c-hero-desc,
+        .mzp-c-hero-tagline {
+            color: transparentize($color-white, .1);
+        }
+    }
+}
+
+.mzp-c-hero-body {
+    margin: 0 auto;
+    max-width: 480px;
+}
+
+.mzp-c-hero-title {
+    @include text-display-lg;
+    margin-bottom: $spacing-md;
+
+    // Product logos
+    .mzp-t-product-firefox & {
+        @include at2x('#{$image-path}/logos/firefox/firefox.png', 80px, 80px);
+    }
+
+    .mzp-t-product-beta & {
+        @include at2x('#{$image-path}/logos/firefox/beta.png', 80px, 80px);
+    }
+
+    .mzp-t-product-developer & {
+        @include at2x('#{$image-path}/logos/firefox/developer.png', 80px, 80px);
+    }
+
+    .mzp-t-product-nightly & {
+        @include at2x('#{$image-path}/logos/firefox/nightly.png', 80px, 80px);
+    }
+
+    .mzp-t-product-focus & {
+        @include at2x('#{$image-path}/logos/firefox/focus.png', 80px, 80px);
+    }
+}
+
+.mzp-c-hero-desc {
+    @include text-body-lg;
+    margin-bottom: $spacing-md;
```

Comment:
Are we abandoning all `$padding` and `$margin` token values? I know they are the same basic value, but if we're going to use something in protocol (or not) I think we should probably make a decision on it and use one thing uniformally. This could be an issue to bikeshed :)

Reply:
Yeah, been meaning to bikeshed this in a meeting but it hasn't come up... I just don't see the benefit of having three different ways to declare the same value. I think we borrowed the separate `$padding-` and `$margin-` tokens idea from another system where they actually equaled different values, e.g. `$padding-sm` was 4px but `$margin-sm` was 8px or something like that. But we're not doing it that way so it just seems redundant in our case.

I can be persuaded if there's a benefit I'm missing, but I think I'd prefer just `$spacing-` for small spaces and `$layout-` for large spaces.

Reply:
Sounds good to me, but we should make sure we file an issue and remove the tokens if everyone agrees 👍